rts/lra/coordinator/pom.xml

```diff
@@ -142,6 +138,10 @@
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
+            <properties>
+                <lra.coordinator.host>localhost</lra.coordinator.host>
```

Comment:
is this needed when there is url?

Reply:
Yes. It's needed as arquillian.xml defines `managementAddress` which is used by arq extension to find where the management can be contacted. This differentiation is needed on ipv4 vs ipv6 here (as it's already defined here in `pom.xml`)